lib/dry/cli/inline.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Dry
+  class CLI
+    require 'dry/cli'
+
+    module Inline
+      extend Forwardable
+
+      AnonymousCommand = Class.new(Dry::CLI::Command)
+
+      delegate %i[desc example argument option] => AnonymousCommand
+
+      def run(arguments: ARGV, out: $stdout)
+        command = AnonymousCommand.new
+        command.define_singleton_method(:call) do |*args|
+          yield(*args)
+        end
+
+        Dry.CLI(command).call(arguments: arguments, out: out)
+      end
+    end
+  end
+end
+
+include Dry::CLI::Inline
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`include` is used at the top level. Use inside `class` or `module`.](https://app.codacy.com/gh/dry-rb/dry-cli/pullRequest?prid=4800973)